api/handler/put.go

```diff
@@ -155,3 +173,22 @@ func parseBasicACL(basicACL string) (uint32, error) {
 		return uint32(value), nil
 	}
 }
+
+func (h *handler) writeError(w http.ResponseWriter, r *http.Request, msg, rid string, err error) {
```

Comment:
Isn't this function too similar to `func (h *handler) registerAndSendError(w http.ResponseWriter, r *http.Request, err error, logText string)`?

Reply:
You're right